chainloader/Makefile

```diff
@@ -92,7 +92,7 @@ flash: flash-$(HOST_OS)
 
 flash-Linux: $(IMAGE_NAME).img
 	mkdir -p $(MNT)
-	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed, expecting card to be already mounted, continuing..."
+	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed, assuming card to be already properly mounted"
```

Comment:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed, assuming card to be mounted properly already"
```

Reply:
If we are getting this pedantic, why not "be properly mounted already"?

Reply:
@BMG93 what's your opinion?

Reply:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed, assuming card to already be properly mounted"
```

Reply:
but shouldn't you stress "properly" and not "mounted"? Because that's the message of this sentence

Reply:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed it has, to already be mounted properly we assume"
```

Reply:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo "Warning: mount failed, assuming card to already be properly mounted on $(MNT)"
```

Reply:
```suggestion
	@mount $(MNT) && echo "mounted SD card to $(MNT)" || echo"Warning: mount failed, assuming card to already be mounted on $(MNT)"
```